OrangeDriver.java

```diff
@@ -0,0 +1,11 @@
+package comp1130;
+
+public class OrangeDriver {
+
+	public static void main(String[] args) {
+		Orange fruit = new Orange ();

```

Comment:
no need to put a space between Orange and () so `new Orange()` would be correct. Otherwise very good!